studio/components/interfaces/TableGridEditor/TableDefinition.tsx

```diff
@@ -75,4 +119,4 @@ const ViewDefinition = ({ name }: ViewDefinitionProps) => {
   )
 }
 
-export default ViewDefinition
+export default observer(ViewDefinition)
```

Comment:
should we still do this? or wrap observer when we declare the const TableDefinition at the beginning?